apps/next/app/dashboard/page.tsx

```diff
@@ -122,11 +170,20 @@ export default async function Page() {
     clerkId: clerkUser.id,
     userInsert,
   });
+  const teamName = `${user.firstName}'s Projects`;
   const userPersonalTeam = await findCreateUserTeam({
-    teamName: `${user.firstName}'s Projects`,
+    teamName,
     userId: user.id,
     userClerkId: clerkUser.id,
   });
+  const { orgId } = auth();
+  if (orgId) {
+    await ensureTeamForClerkOrg({
+      teamName,
+      userId: user.id,
+      clerkOrgId: orgId,
+    });
+  }
```

Comment:
this is much nicer IMO, thanks for doing that

Reply:
Haha. Sure thing.